enhancements/installer/aws-cross-account-dns-zone.md

```diff
@@ -235,9 +277,15 @@ platform dependencies in this config, but it should be carefully considered.
 
 An e2e test will be setup which utilizes both AWS CI accounts.
 
+For the ExternalDNS Operator, we will add e2e tests utilizing both AWS CI accounts using the RoleARN to create DNS
+records.
+
 ### Graduation Criteria
 
-This functionality is targeted for 4.14 GA and for backporting to previous releases. 
+This functionality is targeted for 4.14 GA and for backporting to previous releases.
+
+Note that the External-DNS-Operator updates to support Shared VPC are out of payload and are therefore not aligned with 
+OpenShift releases and will not need backporting.
 
 #### Dev Preview -> Tech Preview
```

Comment:
I suppose the update for ExternalDNS Operator API with the assume role field will not be guarded by a feature gate as it's done for the cluster ingress operator. Probably worth a mention as it may give an impression that the feature gate is still needed.

Reply:
Done.